Adding Defender ATP dependencies doc for clarity (#8968)
MicrosoftDocs/windows-itpro-docs: windows/security/threat-protection/microsoft-defender-atp/deployment-dependencies.md

---
title: Deployment dependencies

> description: Learn about Defender ATP deployment dependencies on each platform OS

Reviewer: "Defender ATP" -> "Defender for Endpoint".

keywords: deploy, prepare, setup, onboard, phase, deployment, deploying, adoption, configuring, dependancies, dependancy
search.product: eADQiWindows 10XVcnh
ms.prod: w10
ms.mktglfcycl: deploy
ms.sitesec: library
ms.pagetype: security
ms.author: jospaid
author: jospaid
ms.localizationpriority: medium
manager: ayhusain
audience: ITPro
ms.collection: 
- M365-security-compliance
- m365solution-endpointprotect
- m365solution-overview  
ms.topic: article
---

> # Deployment dependancies

Reviewer 2: Title typo correction:
```suggestion
# Deployment dependencies
```

> When deploying Defender for Endpoints, each operating systems may require the Defender ATP agent, app, or the Microsoft Monitoring Agent (MMA). The following table outlines the requirements for each platform:

Reviewer 2: Typo corrections:
```suggestion
# Deployment dependencies
When deploying Defender for Endpoint, each operating system may require the Defender ATP agent, app, or the Microsoft Monitoring Agent (MMA). The following table outlines the requirements for each platform:
```

> When deploying Defender for Endpoints, each operating system may require the Defender ATP agent, app, the Microsoft Monitoring Agent (MMA) or some combination of them. The following table outlines the requirements for each platform:

Reviewer: 1. "Defender for Endpoint" is the right term and note "Endpoints".
2. "Defender ATP agent" -> "Defender for Endpoint agent".

Reviewer 2: Thanks. I should have checked the document in more detail.

> |Operating System | Defender Agent/App | MMA Agent | Onboarding Process | 

Reviewer: All the links shouldn't contain "/en-us" to enable localization.

Reviewer 2: @x3nc0n: I can add some change suggestions which you can commit as individual changes, unless you prefer to edit the document and make all the manual changes yourself.

|:-------|:-----|:----|:----|
| Windows 7 SP1 and 8.1 | No | Yes | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
| Windows 10 | No | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
| Windows Server 2008 R2 SP1, 2012 R2, and 2016 | No | Yes | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
| Windows Server 1803 and 2019 | No | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
| macOS | Yes | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
| Linux Server | Yes | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |

> | iOS | Yes | N/A | [App](https://apps.apple.com/us/app/microsoft-defender-atp/id1526737990) |

Reviewer: The link should be to microsoft-defender-atp-ios.md and not to App Store.

| Android | Yes | N/A | [Microsoft Endpoint Manager](https://docs.microsoft.com/en-us/mem/endpoint-manager-overview) |

## See also
- [Phase 3: Onboard](onboarding.md)
- [Plan deployment](deployment-strategy.md)

> - [Defender compatibility](defender-compatibility.md)

Reviewer: This page was deprecated. The right one is microsoft-defender-antivirus-compatibility.md .

---
title: Deployment dependencies
description: Learn about Defender ATP deployment dependencies on each platform OS
keywords: deploy, prepare, setup, onboard, phase, deployment, deploying, adoption, configuring, dependancies, dependancy
search.product: eADQiWindows 10XVcnh
ms.prod: w10
ms.mktglfcycl: deploy
ms.sitesec: library
ms.pagetype: security
ms.author: jospaid
author: jospaid
ms.localizationpriority: medium
manager: ayhusain
audience: ITPro
ms.collection: 
- M365-security-compliance
- m365solution-endpointprotect
- m365solution-overview  
ms.topic: article
---

# Deployment dependancies
## Table of dependancies
When deploying Defender for Endpoints, each operating system may require the Defender ATP agent, app, the Microsoft Monitoring Agent (MMA) or some combination of them. The following table outlines the requirements for each platform:

|Operating System | Defender Agent/App | MMA Agent | Onboarding Process | 
|:-------|:-----|:----|:----|

> Windows 7 SP1 and 8.1 | No | Yes | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
> Windows 10 | No | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
> Windows Server 2008 R2 SP1, 2012 R2, and 2016 | No | Yes | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
> Windows Server 1803 and 2019 | No | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
> macOS | Yes | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
> Linux Server | Yes | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
> iOS | Yes | N/A | [App](https://apps.apple.com/us/app/microsoft-defender-atp/id1526737990) |
> Android | Yes | N/A | [Microsoft Endpoint Manager](https://docs.microsoft.com/en-us/mem/endpoint-manager-overview) |

Reviewer 2: Codestyle/consistency: add leading MD table cell divider + space:
```suggestion
| Windows 7 SP1 and 8.1 | No | Yes | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
| Windows 10 | No | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
| Windows Server 2008 R2 SP1, 2012 R2, and 2016 | No | Yes | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
| Windows Server 1803 and 2019 | No | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
| macOS | Yes | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
| Linux Server | Yes | No | [Options](https://docs.microsoft.com/en-us/windows/security/threat-protection/microsoft-defender-atp/onboard-configure#onboarding-tool-options) |
| iOS | Yes | N/A | [App](https://apps.apple.com/us/app/microsoft-defender-atp/id1526737990) |
| Android | Yes | N/A | [Microsoft Endpoint Manager](https://docs.microsoft.com/en-us/mem/endpoint-manager-overview) |
```

> 
> 
> 

Reviewer 2: Codestyle: no more than 2 consecutive blank lines needed in regular document spacing.
```suggestion

```
